wbstack/src/Settings/LocalSettings.php

```diff
@@ -218,6 +218,18 @@
 $wgPasswordSender = 'noreply@' . getenv('MW_EMAIL_DOMAIN');
 $wgNoReplyAddress = 'noreply@' . getenv('MW_EMAIL_DOMAIN');
 
+// SMTP
+if (getenv('MW_MAILGUN_ENABLED') !== 'yes') {     // do not configure SMTP if Mailgun Extension is wanted
```

Comment:
Do we want this?

Reply:
Looking at the code now (compared to my phab comments) perhaps 2 env vars would be cleaner?